tuf/ngclient/updater.py

```diff
@@ -368,12 +368,15 @@ def _load_snapshot(self) -> None:
             self._trusted_set.update_snapshot(data)
             self._persist_metadata("snapshot", data)
 
-    def _load_targets(self, role: str, parent_role: str) -> None:
+    def _load_targets(self, role: str, parent_role: str) -> Metadata:
```

Comment:
The return type annotation should be the same as the one for `trusted_set.update_delegated_targets` or `Metadata[Targets]`.